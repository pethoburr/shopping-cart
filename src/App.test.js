import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import App from './App';
import About from './components/about.js';
import Shop from './components/shop.js';
import Contact from './components/contact.js';
import Nav from './components/nav.js';
import Home from './components/home.js';
import { MemoryRouter } from 'react-router-dom';

describe('Home page', () => {
  it('renders correct heading', () => {
    render(
      <MemoryRouter initialEntries={['/']} initialIndex={0}>
    <Home />
    </MemoryRouter>
    )
    expect(screen.getByRole('heading').textContent).toMatch(/Bike Life/i);
    expect(screen.getByRole('list').textContent).toMatch(/AboutShop/i);
  })
})

describe('About page', () => {
  it('renders correct heading', () => {
    const props = {
      counter: 0
    }
    render(
      <MemoryRouter initialEntries={['/']} initialIndex={0}>
    <About {...props} />
    </MemoryRouter>
    )
    screen.getAllByText(/OUR MISSION/i)
    screen.getByText(/OUR TEAM/i);
    screen.getByText(/FEATURED/i);
    screen.getByText(/Feeling Lost/i, {exact: false});
  })
})

describe('Shop page', () => {
  it('renders page', async () => {
    const counter = jest.fn();
    const { container } = render(
      <MemoryRouter initialEntries={['/']} initialIndex={0}>
    <Shop count={counter} />
    </MemoryRouter>
    )
    expect(container).toMatchSnapshot();
  })
})

describe('nav page', () => {
  const counter = 0
  it('renders navbar', () => {
    const { container } = render(
      <MemoryRouter initialEntries={['/']} initialIndex={0}>
    <Nav count={counter} />
    </MemoryRouter>
    )
    expect(container).toMatchSnapshot(); 
  })
})