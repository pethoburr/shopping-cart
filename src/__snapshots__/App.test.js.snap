// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shop page renders page 1`] = `
<div>
  <div>
    <nav>
      <a
        href="/"
        style="text-decoration: none;"
      >
        <h3>
          <i
            class="fa-solid fa-motorcycle fa-spin"
          />
          Â Bike Life
        </h3>
      </a>
      <ul>
        <a
          href="/"
          style="text-decoration: none;"
        >
          <li
            class="links"
          >
            Home
          </li>
        </a>
        <a
          href="/about"
          style="text-decoration: none;"
        >
          <li
            class="links"
          >
            About
          </li>
        </a>
        <a
          href="/shop"
          style="text-decoration: none;"
        >
          <li
            class="links"
          >
            Shop
          </li>
        </a>
        <a
          href="/contact"
          style="text-decoration: none;"
        >
          <li>
            <div
              class="cartEl"
            >
              <span
                class="MuiBadge-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                  />
                </svg>
                 
                <span
                  class="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorSecondary MuiBadge-invisible"
                >
                  0
                </span>
              </span>
            </div>
          </li>
        </a>
      </ul>
    </nav>
    <div
      class="shopContainer"
    >
      <h4>
        SHOP
      </h4>
      <div
        class="allPics"
      >
        <div
          class="picContainer"
          id="ducati"
        >
          <img
            alt="2023 ducati V4 R"
            class="ducatiImg"
            src="v4.jpg"
          />
          <p>
            2023 Ducati Panigale V4 R
          </p>
          <p>
            <i>
              MSRP * $52,995
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Kawasaki H2R"
            src="h2r.jpg"
          />
          <p>
            2023 Kawasaki H2R
          </p>
          <p>
            <i>
              MSRP * $66,900
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Kawasaki ZX10R"
            src="zx10r.jpg"
          />
          <p>
            2023 Kawasaki ZX10R
          </p>
          <p>
            <i>
              MSRP * $21,299
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Kawasaki ZX6R"
            src="zx6r.jpg"
          />
          <p>
            2023 Kawasaki ZX6R
          </p>
          <p>
            <i>
              MSRP * $13,999
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Suzuki GSXR1000RR"
            src="gsxr1000.jpeg"
          />
          <p>
            2023 Suzuki GSR1000RR
          </p>
          <p>
            <i>
              MSRP * $23,999
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Suzuki GSXR750RR"
            class="gsxrImg"
            src="gsxr750.jpeg"
          />
          <p>
            2023 Suzuki GSXR750RR
          </p>
          <p>
            <i>
              MSRP * $13,899
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Suzuki GSXR600RR"
            src="gsxr600.jpg"
          />
          <p>
            2023 Suzuki GSXR600RR
          </p>
          <p>
            <i>
              MSRP * $12,899
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Honda CBR1000RR"
            src="cbr1000rr.webp"
          />
          <p>
            2023 Honda CBR1000RR
          </p>
          <p>
            <i>
              MSRP * $37,290
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Honda CBR600RR"
            class="cbrImg"
            src="cbr600rr.jpeg"
          />
          <p>
            2023 Honda CBR600RR
          </p>
          <p>
            <i>
              MSRP * $17,701
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Yamaha R1"
            src="r1.jpeg"
          />
          <p>
            2023 Yamaha R1
          </p>
          <p>
            <i>
              MSRP * $22,799
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Yamaha R6"
            src="r6.jpg"
          />
          <p>
            2023 Yamaha R6
          </p>
          <p>
            <i>
              MSRP * $15,859
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 BMWS1000RR"
            class="bmwImg"
            src="bmw.jpeg"
          />
          <p>
            2023 BMWS1000RR
          </p>
          <p>
            <i>
              MSRP * $20,395
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="picContainer"
        >
          <img
            alt="2023 Aprilla RSV4"
            src="aprilla.avif"
          />
          <p>
            2023 Aprilla RSV4
          </p>
          <p>
            <i>
              MSRP * $27,995
            </i>
          </p>
          <div
            class="MuiButtonGroup-root"
            role="group"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13H5v-2h14v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`nav page renders navbar 1`] = `
<div>
  <nav>
    <a
      href="/"
      style="text-decoration: none;"
    >
      <h3>
        <i
          class="fa-solid fa-motorcycle fa-spin"
        />
        Â Bike Life
      </h3>
    </a>
    <ul>
      <a
        href="/"
        style="text-decoration: none;"
      >
        <li
          class="links"
        >
          Home
        </li>
      </a>
      <a
        href="/about"
        style="text-decoration: none;"
      >
        <li
          class="links"
        >
          About
        </li>
      </a>
      <a
        href="/shop"
        style="text-decoration: none;"
      >
        <li
          class="links"
        >
          Shop
        </li>
      </a>
      <a
        href="/contact"
        style="text-decoration: none;"
      >
        <li>
          <div
            class="cartEl"
          >
            <span
              class="MuiBadge-root"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                />
              </svg>
               
              <span
                class="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorSecondary MuiBadge-invisible"
              >
                0
              </span>
            </span>
          </div>
        </li>
      </a>
    </ul>
  </nav>
</div>
`;

exports[`nav page renders page 1`] = `
<div>
  <nav>
    <a
      href="/"
      style="text-decoration: none;"
    >
      <h3>
        <i
          class="fa-solid fa-motorcycle fa-spin"
        />
        Â Bike Life
      </h3>
    </a>
    <ul>
      <a
        href="/"
        style="text-decoration: none;"
      >
        <li
          class="links"
        >
          Home
        </li>
      </a>
      <a
        href="/about"
        style="text-decoration: none;"
      >
        <li
          class="links"
        >
          About
        </li>
      </a>
      <a
        href="/shop"
        style="text-decoration: none;"
      >
        <li
          class="links"
        >
          Shop
        </li>
      </a>
      <a
        href="/contact"
        style="text-decoration: none;"
      >
        <li>
          <div
            class="cartEl"
          >
            <span
              class="MuiBadge-root"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                />
              </svg>
               
              <span
                class="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorSecondary MuiBadge-invisible"
              >
                0
              </span>
            </span>
          </div>
        </li>
      </a>
    </ul>
  </nav>
</div>
`;
